autopush/integration/app/converter.js

```diff
@@ -1,24 +1,21 @@
-exports.rgbToHex = function(red, green, blue) {
-
-  var redHex   = red.toString(16);
-  var greenHex = green.toString(16);
-  var blueHex  = blue.toString(16);
-
-  return pad(redHex) + pad(greenHex) + pad(blueHex);
-
+var converter = {
+  rgbToHex: function (red, green, blue) {
+    var redHex   = red.toString(16);
+    var greenHex = green.toString(16);
+    var blueHex  = blue.toString(16);
+
+    return pad(redHex) + pad(greenHex) + pad(blueHex);
+  },
+
+  hexToRgb: function (hex) {
+    var red   = parseInt(hex.substring(0, 2), 16);
+    var green = parseInt(hex.substring(2, 4), 16);
+    var blue  = parseInt(hex.substring(4, 6), 16);
+
+    return [red, green, blue];
+  }
```

Comment:
Since this code will be tested in the browser, we can't use Node.js' `require()` and `exports` syntax.
Therefore, everything will be set as `var` and exported as globals.